src/map/script.c

```diff
@@ -25792,10 +25792,10 @@ static void script_parse_builtin(void)
 
 		BUILDIN_DEF(itempreview, "i"),
 		BUILDIN_DEF(enchantitem, "iii"),
-		BUILDIN_DEF(expandInventoryAck, "i?"),
```

Comment:
Probably wrong names better add with BUILDIN_DEF_DEPRECATED?

Reply:
huh ???
you are the one that made the commit ...
https://github.com/HerculesWS/Hercules/blob/master/npc/other/inventory_expansion.txt
and it used in this script, why deprecate it ?

Reply:
@AnnieRuru , 4144 probably meant that deprecate commands with wrong names (camel case words) and add new lines for correct name.

Reply:
erm ... this script command was just implemented recently ... right ?
its not like this script command already wide-spread ...

Reply:
near 2 months i think. merged to hercules 1 month ago

Reply:
hmm ... some in-game testing
```
-	script	khfsdkhjsf	FAKE_NPC,{
OnPCLoginEvent:
	debugmes getInventorySize +"";
}
```
already throw error like
```
[Error]: script_add_str: detected possible use of wrong case in a script. Found
'getInventorySize', probably meant to be 'getinventorysize' (in 'npc/zzz.txt').
```
you sure still need to fix this ?

Reply:
1 month++ bump

I have tested this PR will throw error on **parse time**, with the error shown in above post
@4144

Reply:
please decrecated all commands with wrong case.
for example: ``BUILDIN_DEF2_DEPRECATED(expandinventoryack, "expandInventoryAck", "i?")``

Reply:
nonono, please test this PR, this doesn't need to deprecate
there is already the wrong case detection in the script engine to throw that error already

also, when I tested the `BUILDIN_DEF2_DEPRECATED` like you said, when I did it before
the small letter case got deprecated as well for some reason ...

try it yourself